src/main.c

```diff
@@ -2906,15 +2906,18 @@ int sqlite3_limit(sqlite3 *db, int limitId, int newLimit){
 */
 int sqlite3ParseUri(
   const char *zDefaultVfs,        /* VFS to use if no "vfs=xxx" query option */
+  const char *zDefaultWal,        /* WAL module to use if no "wal=xxx" query option */
```

Comment:
It seems that the comment on the function is now out of date. It could be a good idea to update it with some description of the new params.

> The second argument contains the URI (or non-URI filename)
** itself